docs/config/ganache.toml

```diff
@@ -20,7 +20,9 @@ propagation_timeout = 2000
 
 fetch_timeout = 5000
 all_campaigns_timeout = 5000
-channel_tick_timeout = 5000
+# for test_harness make it larger
+# Default: 5000
+channel_tick_timeout = 8000
```

Comment:
I still get timeouts in test_harness sometimes, we have to check why this is - performance issue or something wrong with the services